.github/workflows/new-pull-request.yml

```diff
@@ -0,0 +1,37 @@
+name: add comment
+
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  checklist:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/github-script@v6
+      with:
+        script: |
+          github.rest.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: `Thank you for contributing! ✨ PlasmaPy's future depends on contributors like you, so we deeply appreciate it! 🌱 Please check out the [**contributor guide**](https://docs.plasmapy.org/en/latest/contributing/index.html) for more information on how to contribute.
+
+            The following checklist will help you and the reviewer through the code review process.
+             - Overall
+               - [ ] Does the PR do what it intends to do?
+               - [ ] Except for very minor changes, is a changelog entry included and consistent with the [changelog guide](https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html)?
+               - [ ] Are the continuous integration checks passing? (Most linter problems can be automagically fixed by commenting on this PR with \`pre-commit.ci autofix\`.)
```

Comment:
The problem prior to this was that the backticks were used to begin and end this text block, so I needed to use an escape character here.